README.md

```diff
@@ -1,22 +1,56 @@
-# Canonical livepatch machine charm
+# Canonical Livepatch Server (Machine Charm)
 
-The livepatch machine charm is the easiest and the recommended way to deploy livepatch. This charms installs and configures the Ubuntu Linux Livepatching Utility and Daemon. Canonical Livepatch patches high and critical linux kernel vulnerabilities removing the immediate need to reboot to upgrade the kernel, instead allowing the downtime to be scheduled. It is a part of the Ubuntu Pro offering.
+[![CharmHub Badge](https://charmhub.io/canonical-livepatch-server/badge.svg)](https://charmhub.io/canonical-livepatch-server)
+[![Release](https://github.com/canonical/livepatch-machine-charm/actions/workflows/publish_charm.yaml/badge.svg)](https://github.com/canonical/livepatch-machine-charm/actions/workflows/publish_charm.yaml)
+[![Tests](https://github.com/canonical/livepatch-machine-charm/actions/workflows/test.yaml/badge.svg?branch=main)](https://github.com/canonical/livepatch-machine-charm/actions/workflows/test.yaml?query=branch%3Amain)
+
+## Description
+
+The Livepatch machine charm is the easiest and the recommended way to deploy Livepatch. This charm installs and configures the Ubuntu Linux Livepatch-ing utility and daemon. Canonical Livepatch patches high and critical linux kernel vulnerabilities, removing the immediate need to reboot to upgrade the kernel,
+instead allowing the downtime to be scheduled. It is a part of the Ubuntu Pro offering.
+
+⚠️ For users who want to deploy an entire Livepatch on-prem server (including its dependencies), it is recommended to use the [bundle](https://charmhub.io/canonical-livepatch-onprem) made for this purpose. For more detailed steps on using the bundle, please see the [tutorials](https://ubuntu.com/security/livepatch/docs/livepatch_on_prem/tutorial) on the Livepatch website.
```

Comment:
We should probably update the description to point out that the bundle for the operator charm is available in the `machine` branch rather than the `default` branch.

Reply:
Done.